README.md

```diff
@@ -35,7 +35,11 @@ metadata:
 
 1. _My deployment has been shut down and won't scale up until tomorrow, what options do I have to bring it back up?_
 
-    Re-triggering the pipeline for deploying it again would make it available until the next scale-down window.
+    Push a new commit that changes the helm chart (if the docker image is re-created at every push a dummy commit is enough). This only works if `.Values.scale.enabled=false` and `.Values.replicas` is set.
```

Comment:
Just for my understanding, in our context, if scale is disabled, replicas is always set, correct? It would take the default value.